### ISSUE TITLE
Burp 1.7 new added default tabs

### ISSUE THREAD

**Issue by mgeeky** [hvqzao/burp-wildcard#1]
Hello there,

I'm using your burp-wildcard plugin which gets it's simple job fine. But unfortunately, since Burp has updated their UI interface in 1.7 version - they have separated `Options` tab into two of them: `Project Options` and `User Options`. Therefore, your `burpTabs` array located at: `BurpExtender.java:59` is no longer relevant and should get updated :)

Could you update your extension?

**Comment by hvqzao** [comment]
Hi mgeeky,

I'm always glad to hear when someone finds this extension handy!
I've just added compatibility with Burp Pro 1.7.
Download link:

https://github.com/hvqzao/burp-wildcard/releases/download/v1.03/wildcard.jar

**Comment by mgeeky** [comment]
Hi,

Thanks for your update. Possibly silly question: have you tested this update?
Right now my extender is reporting the following exception while loading your `wildcard.jar`:

```
java.lang.ClassNotFoundException: burp.BurpExtender
    at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
    at java.lang.Class.forName0(Native Method)
    at java.lang.Class.forName(Class.java:348)
    at burp.xob.a(Unknown Source)
    at burp.xob.<init>(Unknown Source)
    at burp.v0b.a(Unknown Source)
    at burp.ecb.run(Unknown Source)
    at java.lang.Thread.run(Thread.java:745)
```

Any ideas what is going on? :)

**Comment by hvqzao** [comment]
Hmm... Yup, I've tested it on 1.7beta pro (temporary project, defaults on Kali Linux) and 1.6 free on Windows. Now I've verified it on different platforms (1.6.35 free / 1.6.39 pro / 1.7beta pro) and everything seems to work properly.

![wildcard-1 03](https://cloud.githubusercontent.com/assets/4956006/14605889/ffd6ad82-057a-11e6-9aa5-2f089ff3c9bb.png)

and Errors:

![wildcard-1 03 1](https://cloud.githubusercontent.com/assets/4956006/14605957/4e2c9320-057b-11e6-95ba-34784e45c886.png)

Try to remove extension, exit burp, re-download new jar, put it in some place, start burp and add this extension. I tried both - removing old one + adding new one and just replacing wildcard.jar file and it worked every time. Which Java are you using? Oracle? 1.8? 64bit?
